### PR TITLE
Avatar component

### PR DESCRIPTION
**Pull request by gouravmarch20**
avatar componet : html code tested

### CODE REVIEW

**Comment by gouravmarch20** [comment]
not completed css

**Comment by gouravmarch20** [comment]
done

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <title>Document</title>
 </head>
 <body>
+    <h2>master</h2>
```
Comment (parthgourav) [inline index.html:10]: looks good

Comment (parthgourav) [reply index.html:10]: need semetic text & css

**Pull request merged**